### PR TITLE
Add support for dependabot_alert webhook event

### LINKED ISSUE #2790: dependabot_alert Event missing in eventTypeMapping
Trying to use this package to parse events, but `dependabot_alert` is not included.

Ref: https://docs.github.com/en/webhooks-and-events/webhooks/webhook-events-and-payloads#dependabot_alert

### PR DESCRIPTION
**Pull request by nikpivkin**
Closes #2790 

Added support for [dependabot_alert](https://docs.github.com/en/webhooks-and-events/webhooks/webhook-events-and-payloads#dependabot_alert) webhook event.

### CODE REVIEW

File: github/event_types.go [hunk 1]
```diff
@@ -134,6 +134,22 @@ type DeleteEvent struct {
 	Installation *Installation `json:"installation,omitempty"`
 }
 
+// DependabotAlertEvent is triggered when there is activity relating to Dependabot alerts.
+// The Webhook event name is "dependabot_alert".
+//
+// GitHub API docs: https://docs.github.com/en/webhooks-and-events/webhooks/webhook-events-and-payloads#dependabot_alert
+type DependabotAlertEvent struct {
+	Action *string         `json:"action,omitempty"`
+	Alert  DependabotAlert `json:"alert"`
```
Comment (gmlewis) [inline github/event_types.go:143]:
```suggestion
	Alert  *DependabotAlert `json:"alert,omitempty"`
```

File: github/event_types.go [hunk 2]
```diff
@@ -134,6 +134,22 @@ type DeleteEvent struct {
...
+
+	// The following field is only present when the webhook is triggered on
+	Installation *Installation `json:"installation,omitempty"`
+	Organization *Organization `json:"organization,omitempty"`
+	Enterprise   *Enterprise   `json:"enterprise,omitempty"`
+	Repo         *Repository   `json:"repository,omitempty"`
+	Sender       *User         `json:"sender,omitempty"`
```
Comment (gmlewis) [inline github/event_types.go:150]:
```suggestion
	// The following fields are only populated by Webhook events.
	Installation *Installation `json:"installation,omitempty"`
	Enterprise   *Enterprise   `json:"enterprise,omitempty"`
	Repo         *Repository   `json:"repository,omitempty"`
	Sender       *User         `json:"sender,omitempty"`

	// The following field is only present when the webhook is triggered on
	// a repository belonging to an organization.
	Organization *Organization `json:"organization,omitempty"`
```

**Review by gmlewis — CHANGES_REQUESTED** [review]
Thank you, @nikpivkin !

**Review by gmlewis — APPROVED** [review]
Thank you, @nikpivkin !
LGTM.

Awaiting second LGTM+Approval from any other contributor to this repo before merging.

**Review by WillAbides — APPROVED** [review]

**Comment by gmlewis** [comment]
Thank you, @WillAbides !
Merging.

**Pull request merged**